src/device_imxrt.c

```diff
@@ -0,0 +1,278 @@
+#include "device.h"
+#include "log.h"
+
+#if MG_DEVICE == MG_DEVICE_RT1020 || MG_DEVICE == MG_DEVICE_RT1060
+
+#include <flexspi.h>  /**************************************************************************************************************************/
```

Comment:
This is going to be a heck of a problem...
There is quite likely no file with that name in Xpresso, and probably file contents I got from several files and built just one...
This is one of the reasons why I think this has to be part of Mongoose.
@cpq ?

Some clashing may also occur with Xpresso, for the same reasons in the comment just bellow this line

Reply:
This include looks dangerous to me.
Generally, we do not include any device-specific files from an SDK. For two reasons:
1. To be independent from any 3rd party source
2. To protect from any possible clashes